src/main/java/com/manardenza/controller/UserController.java

```diff
@@ -1,7 +1,7 @@
 package com.manardenza.controller;
 
 import com.manardenza.entity.User;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.manardenza.entity.User'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)